### PR TITLE
Add missing UserID key in openstack keypairs configuration

### PR DESCRIPTION
**Pull request by weinimo**

### CODE REVIEW

File: pkg/octavia/amphora_ssh.go [hunk 1]
```diff
@@ -35,6 +35,7 @@ import (
 	corev1 "k8s.io/api/core/v1"
 	k8serrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"github.com/openstack-k8s-operators/octavia-operator/pkg/octavia"
```
Comment (gthiemonge) [inline pkg/octavia/amphora_ssh.go:38]: this is the current package, no need to import it

**Review by gthiemonge — COMMENTED** [review]
We probably also need to check that the "existing" key belongs to the right tenant

**Comment by gthiemonge** [comment]
proposed an update at https://github.com/openstack-k8s-operators/octavia-operator/pull/276/files

**Comment by weinimo** [comment]
superseded by #276

**Pull request closed without merging**